download_asfserial_rsmas.py

```diff
@@ -37,13 +59,14 @@ def run_download_asf_serial():
 	"""
 	
 	status = subprocess.Popen(['download_ASF_serial.py', '-username', password.asfuser, '-password', password.asfpass, 'new_files.csv']).wait()
-	
+	logger.log(logging.INFO, status)
 	return status
 
 if __name__ == "__main__":
-	
+	logger.log(logging.INFO, "DATASET: %s", str(inps.template.split('/')[-1].split(".")[0]))
```

Comment:
Does this code run without error? It looks like `inps` has value `None` when this code runs.

Reply:
Good catch. Might have been fixed already.